QUARKUS-4430.md

```diff
@@ -0,0 +1,35 @@
+# QUARKUS-3535 - Add a metric for rejected submissions to the pool in RHBQ 3.2
+
+JIRA: https://issues.redhat.com/browse/QUARKUS-4430
+
+PR: https://github.com/quarkusio/quarkus/pull/39583
+
+Upstream issue: https://github.com/quarkusio/quarkus/issues/39581
+
+Verify that URIs in metrics tags are correctly recorded when they match a client or server pattern,
```

Comment:
my understanding is that this change is about path templates, so having test coverage for 302 and 404 is cool, but please stress in the TP what is it that you want to test, e.g. `/moved/{id}`. because you need to create a test coverage that pass with 3.8.5 and but fails with 3.8.4. thanks

now, I understand that this text comes from the PR title `Keep the URIs in the metrics tag if they match a client or server pattern`, but I am not convinced it's correct term. It is path pattern in either client or server path.